### PR TITLE
Fix issue #462 where PropertyCallbackInfo data is not stored safely.

### PR DESCRIPTION
**Pull request by jjrv**

### CODE REVIEW

**Comment by kkoopa** [comment]
LGTM if CI is happy.

File: nan_callbacks_pre_12_inl.h [hunk 1]
```diff
@@ -149,7 +149,7 @@ class FunctionCallbackInfo {
 template<typename T>
 class PropertyCallbackInfoBase {
   const v8::AccessorInfo &info_;
-  const v8::Local<v8::Value> &data_;
+  const v8::Local<v8::Value> data_;
 
  public:
   explicit inline PropertyCallbackInfoBase(
```
Comment (kkoopa) [inline nan_callbacks_pre_12_inl.h]: Update the constructor here to take a Local and not a reference. Same goes for superclasses.

**Comment by jjrv** [comment]
Better like this? There's still some Set and Reset functions remaining that take a reference to v8::Local. But I didn't find any cases where it would store such a reference except that single first one.

**Comment by kkoopa** [comment]
It looks fine at a glance. These constructors should never be directly used by user code anyway, so changing them is fine. Set and Reset functions should take references, as that is (supposed to be) part of the emulated V8 API.

**Comment by kkoopa** [comment]
Merged as 55f50adedd543098526c7b9f4fffd607d3f9861f fefd813d432a5368a87da61eae48587b0f9e874b

**Pull request closed without merging**